homework/config-files/hackyourtemperature/server.js

```diff
@@ -0,0 +1,28 @@
+"use strict"
+
+const express = require('express')
+
+const app = express()
+
+app.use(express.json())
+app.use(express.urlencoded({extended:false}))
+
+app.get('/', (req,res) => {
+  res.setHeader("Content-type","text/plain")
+  res.send("hello from backend to frontend!")
+})
+
+app.post('/weather',(req,res) => {
+  const city = req.body.cityName
+  if(!city){
+    return res.status(400).json({msg:'City not found!'})
+  }
```

Comment:
Woohoo! Nice error handling 👏 

```suggestion
  const city = req.body.cityName
  if(!city){
    return res.status(400).json({msg:'City not provided!'})
  }
```

Although I personally would be a bit more clear with the error. It's not that the city wasn't found, it just wasn't sent!